processing/data_collection/gazette/spiders/sc_lages.py

```diff
@@ -0,0 +1,49 @@
+
+from dateparser import parse
+from datetime import datetime
```

Comment:
Reorder the imports following PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)